eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1073,7 +1073,7 @@ protected function assertSubtreeProperties(array $expectedValues, Location $loca
         $repository = $this->getRepository();
         $locationService = $repository->getLocationService();
 
-        if ($location->id === $stopId) {
+        if ($location->id == $stopId) {
```

Comment:
Yes, correct. I'll leave this as is and add cast on the other PR, though anyone relying on ID being `int` would be against what we prescribe.